qa/README.md

```diff
@@ -80,25 +92,24 @@ public class ProcessDefinitionQueryTest {
 
 **Need:**
 
-* We need to configure the broker for specific authentication
-* We need to make use of the `@RegisterExtension` to pass in the Broker, pre-configured
-* We need to make sure that the test is tagged with `@Tag("multi-db-test")`
+* We need to configure the broker for specific authentication, for that we need to make use of `TestStandaloneBroker` or `TestSimpleCamundaApplication`
+* We need to annotate the test applications with `MultiDbTestApplication`, to mark application managed by `CamundaMultiDbExtension`
 
 **Optional:**
 
 * We might not want to run the test for all secondary storage, as it is not yet supported. We can use the `@DisabledIfSystemProperty` annotation
+* We might want to disable the lifecycle management of the annotated test application, this can be done via: `@MultiDbTestApplication(managedLifecycle = false)`
+  * This is especially useful, if we want to start and stop the test application inside the test (to validate restarts, or something similar)
```

Comment:
```suggestion
* We might want to disable the lifecycle management of the annotated test application. This can be done via `@MultiDbTestApplication(managedLifecycle = false)`.
  * This is especially useful if we want to start and stop the test application inside the test (to validate restarts or something similar)
```